### ISSUE TITLE
Help for Code

### ISSUE THREAD

**Issue by AlfaHughes** [NiccoloCavagnero/EntropicScore#2]
I'm truly impressed by the Entropy Score Based NAS that your project has achieved.
I will really appreciate it if you open the code source

**Comment by NiccoloCavagnero** [comment]
Thank you for your interest. I am not planning to release the code for now, however the metric computation is straightforward:
1) replace norm layers such that their output is in [0,1], by dividing for the channel-wise maximum;
2) for each normalised activation, compute   - torch.mean(activation * torch.log(activation + epsilon)).
3) sum the scores across the different activations.

**Comment by AlfaHughes** [comment]
Thank you  for sharing the details of the metric computation. I believe it has the potential to significantly contribute to the field, and I am eager to see how it evolves. Good luck!

**Comment by NiccoloCavagnero** [comment]
Thank you again for your kind words, do not esitate to contact me if I can be helpful.
Good luck with your work!
